juicebox_scripts/juicebox_assembly_converter.py

```diff
@@ -497,9 +497,9 @@ def _make_scaffold_name(self, index, scaffold):
                 scaffold_length += int(contig[1])
                 if contig != scaffold[-1]:
                     scaffold_length += self.gap_size
-            scaffold_name = 'PGA_scaffold_{0} {1}_contigs length_{2}'.format(index,
-                                                                             contig_count,
-                                                                             scaffold_length)
+            scaffold_name = 'PGA_scaffold_{0}__{1}_contigs__length_{2}'.format(index,
```

Comment:
is there a reason for "__" rather than "_"? something to do with fasta format?

Reply:
It's so you can split on "__" and get the tokens back separately. It's not a common convention, but it was something I first saw in the original LACHESIS FASTA outputs, and we have followed suit with ours. Zev suggested switching to spaces instead of __ as part of revamping this script, but we got some feedback from an early user of this script that it made some things harder with samtools faidx. Since we didn't change to spaces anywhere else, I think we should just change back to __ here